pkg/apis/clusterlink.net/v1alpha1/import.go

```diff
@@ -65,6 +65,8 @@ type ImportSpec struct {
 	// +kubebuilder:default="round-robin"
 	// LBScheme is the load-balancing scheme to use (e.g., random, static, round-robin)
 	LBScheme LBScheme `json:"lbScheme"`
+	// DnsName is an optional external dns name for this imported service
+	DnsName string `json:"dnsName,omitempty"`
```

Comment:
Based on Go's convention this should be `DNSName`

```suggestion
	// DNSName is an optional external DNS name for this imported service
	DNSName string `json:"dnsName,omitempty"`
```

Reply:
Also, `Alias` might be a better name since we create an alias for the Service (it has a DNS name given by the Service and k8s)

Reply:
Updated to Alias.